server/util/getMetrics.js

```diff
@@ -78,6 +78,53 @@ const countUnexpectedBehaviors = ({
     return countScenarioResult(scenarioResult);
 };
 
+const countUnexpectedBehaviorsSeverity = (
```

Comment:
Probably a future enhancement - would it be possible to combine the `getMetrics` from the client side and server side into one util?

Reply:
Perfect!